AppBuilder/platform/dataFields/ABFieldTree.js

```diff
@@ -1,228 +1,18 @@
-                     });
-
-                     tree.unblockEvent();
-                  } else {
-                     tree.blockEvent(); // prevents endless loop
-                     this.data.eachSubItem(id, function (obj) {
-                        if (obj.id != id) tree.uncheckItem(obj.id);
-                     });
-                     tree.unblockEvent();
-                  }
-
-                  // var rowData = {};
-                  // rowData[field.columnName] = $$(idTree).getChecked();
-                  //
-                  // field.setValue($$(parentComponent.ui.id), rowData);
-               },
-            },
-         },
-      ];
-   },
-
-   // defaultValues: the keys must match a .name of your elements to set it's default value.
-   defaultValues: ABFieldTreeCore.defaultValues(),
-
-   // rules: basic form validation rules for webix form entry.
-   // the keys must match a .name of your .elements for it to apply
-   rules: {},
-
-   // include additional behavior on default component operations here:
-   // The base routines will be processed first, then these.  Any results
-   // from the base routine, will be passed on to these:
-   logic: {
-      clear: (ids) => {
-         $$(ids.options).clearAll();
-         $$(ids.options).config.height = 0;
-         $$(ids.options).resize();
-         $$(ids.options).refresh();
-      },
-
-      populate: (ids, field) => {
-         treeCol.clearAll();
-         treeCol.parse(field.settings.options);
-         $$(ids.options).openAll();
-         $$(ids.options).config.height = $$(ids.options).count() * 28 + 18; // Number of pages plus 9px of padding top and bottom
-         $$(ids.options).resize();
-         $$(ids.options).refresh();
-      },
-
-      values: (ids, values) => {
-         // Get options list from UI, then set them to settings
-         values.settings.options = treeCol.serialize();
-         return values;
-      },
-
-      show: (ids) => {
-         $$(ids.options).data.sync(treeCol);
-      },
-   },
-
-   // perform any additional setup actions here.
-   // @param {obj} ids  the hash of id values for all the current form elements.
-   // it should have your elements + the default Header elements:
-   // .label, .columnName, .fieldDescription, .showIcon
-   init: (ids) => {},
-});
+const L = (...params) => AB.Multilingual.label(...params);
 
 module.exports = class ABFieldTree extends ABFieldTreeCore {
-   // constructor(values, object) {
-   //    super(values, object);
-   // }
-
-   /*
-    * @function propertiesComponent
-    *
-    * return a UI Component that contains the property definitions for this Field.
-    *
-    * @param {App} App the UI App instance passed around the Components.
-    * @param {stirng} idBase
-    * @return {Component}
-    */
-   static propertiesComponent(App, idBase) {
-      return ABFieldTreeComponent.component(App, idBase);
+   constructor(values, object) {
```

Comment:
this constructor doesn't look like it's adding anything, so lets just comment it out.